app/uk/gov/hmrc/economiccrimelevyreturns/views/ViewUtils.scala

```diff
@@ -60,10 +60,8 @@ object ViewUtils {
   def formatToday(translate: Boolean = true)(implicit messages: Messages): String =
     formatLocalDate(LocalDate.now(UkZoneId), translate)
 
-  def formatMoney(amount: Number): String = {
-    val formatter = NumberFormat.getCurrencyInstance(Locale.UK)
-    formatter.format(amount)
-  }
+  def formatMoney(amount: BigDecimal): String =
```

Comment:
Couple of questions:

Are there any unit tests for this? If not worth adding some

Does this format the number to contain commas for readability?

Reply:
I'll add some unit tests. The format string includes the "," modifier so that numbers are formatted with the numeric thousands separator character (depending on the system locale).